Makefile

```diff
@@ -415,3 +415,16 @@ test-app-dev: tmp/.ekam-run
 	@cp src/sandstorm/test-app/test-app.capnp tmp/sandstorm/test-app/test-app.capnp
 	@cp src/sandstorm/test-app/*.html tmp/sandstorm/test-app
 	spk dev -Isrc -Itmp -ptmp/sandstorm/test-app/test-app.capnp:pkgdef
+
+# ====================================================================
+# meteor-testapp.spk
+
+meteor-spk-0.3.2/meteor-spk:
+	@$(call color,downloading meteor-spk)
+	@curl https://dl.sandstorm.io/meteor-spk-0.3.2.tar.xz | tar Jxf -
+
+tests/assets/meteor-testapp.spk: meteor-testapp meteor-spk-0.3.2/meteor-spk $(shell find meteor-testapp -type f -o -type d)
+	@cd meteor-testapp && PATH="$$PATH:$$PWD/bin" && ../meteor-spk-0.3.2/meteor-spk pack ../tests/assets/meteor-testapp.spk
```

Comment:
You probably need to set the PATH before changing directories, otherwise PWD will have the wrong value (since it will point at the meteor-testapp directory). You should also put `$PWD/bin` at the front of the list rather than the back so that it gets searched first.

    @PATH="$$PWD/bin:$$PATH" && cd meteor-testapp && ../meteor-spk-0.3.2/meteor-spk pack ../tests/assets/meteor-testapp.spk